keyboards/doro67/multi/readme.md

```diff
@@ -2,6 +2,13 @@
 
 65% custom keyboard made by 80ultraman/Alf/Backprop Studios with multiple layout support. Despite the layout options available, layout is dictated by the selected integrated plate.  
 
+**THIS IS THE KEYMAP DIRECTORY FOR NON-CHINA BUYERS**
```

Comment:
```suggestion
**THIS IS THE MULTI-LAYOUT PCB DIRECTORY FOR NON-CHINA BUYERS**
```

Reply:
Non China buyers sounds like if you're not Chinese you can use this lol. 

Maybe

```suggestion
**MULTI-LAYOUT PCB (AVAILABILITY: CHINA + INTERNATIONAL GB)**
```